package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.2.8",
   "description": "A module that parses a string as regular expression and returns the parsed value.",
   "main": "lib/index.js",
+  "typings": "typings/regex-parser.d.ts",
```

Comment:
I'm not good in TS typings: should this file be published on `npm`?

If so, move the `typings` directory into `lib`. Because `lib` is whitelisted.

Reply:
yep it should be published on npm. I'll move the folder an update this PR